samples/akka-sample-sharding-scala/killrweather-fog/src/main/scala/sample/killrweather/fog/WeatherStation.scala

```diff
@@ -57,11 +56,11 @@ private class WeatherStation(context: ActorContext[WeatherStation.Command], wsid
   private val stationUrl = s"http://${settings.host}:${httpPort}/weather/$wsid"
 
   def running: Behavior[WeatherStation.Command] = {
-    context.log.infoN(s"Started WeatherStation {} of total {} with weather port {}",
```

Comment:
@johanandren Isn't this what you refer to?

Reply:
Ah indeed, sorry, everything is good then 👍